docs/extensibility/resources.md

```diff
@@ -13,9 +13,9 @@ Busola supports only the current version of the configuration and the prior one.
 
 Therefore, whenever a new version of the configuration is proposed, you can migrate your configuration to the latest version. To do so, go to your Extension and click the **Migrate** button.
 
-## general section
+## _general_ section
```

Comment:
I know I suggested code font first, but I now think that italics are better, because they still distinguish it from natural text, and thus do their job, but they're not in contradiction with our guidelines.